benchmarks/Microsoft.AspNetCore.Mvc.Performance/ValidationVisitorModelWithValidatedProperties.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using BenchmarkDotNet.Attributes;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+
+namespace Microsoft.AspNetCore.Mvc.Performance
+{
+    public class ValidationVisitorModelWithValidatedProperties : ValidationVisitorBenchmarkBase
+    {
+        private class Person
+        {
+            [Required]
+            public int Id { get; set; }
+
+            [Required]
+            [StringLength(20)]
+            public string Name { get; set; }
+
+            public string Description { get; set; }
+        }
```

Comment:
I'd suggest making this a bit more complicated. Maybe include an array of `Car[]` or something